web3swift/src/Account/EthereumKeyStorage.swift

```diff
@@ -9,28 +9,66 @@
 import Foundation
 
 public protocol EthereumKeyStorageProtocol {
```

Comment:
I think we should leave this protocol as-is, and introduce `EthereumMultipleKeyStorageProtocol`

Can even derive an `EthereumKeyStorageProtocol` implementation from a 'multiple' one by passing in an address to use, so it's easily bridgeable with `EthereumAccount`, which only cares about a single key.